src/flask_container_scaffold/util.py

```diff
@@ -9,8 +9,8 @@ def load_yaml(filename='config.yml', logger=None):
     Convenience wrapper around toolchest.yaml::parse to allow you to parse a
     file by path+name
 
-    :param filename: A yaml file to be parsed
-    :param logger: Optional logger for potential errors
+    :param str filename: A yaml file to be parsed
+    :param logger logger: Optional logger for potential errors
```

Comment:
I would have capitalised the first logger since it's a Class, but that's pretty minor.